lib/hyacinth/digital_object/type_def/base.rb

```diff
@@ -37,6 +37,13 @@ def public_writer
           self # always return self to allow for chained calls
         end
 
+        # Constraint is checked by call to include?, so a values hash is verifying by key presence
+        def constrained_to(values = [])
```

Comment:
Going along with my earlier comment, I think we'd want to remove this in favor of `DigitalObjectConcerns::Validations`.

Reply:
It also doesn't protect against `instance_variable_set`, but a validation would.

Reply:
Now I'm good with keeping constraints and going along with the validation method we talked about. 👍